server/logs/lib.lua

```diff
@@ -0,0 +1,86 @@
+Logs = {}
+Logs.apiUrl= "https://api.fivemanagee.com/logs"
```

Comment:
URL is spelled wrong, also I'd probably make these their own local variables so they can't be mutated elsewhere.

Reply:
Agree